mantidproject/mantid
# Engg::Enabled Multi-Run Focusing for Focus Tab with Two Different Focusing Modes

fixes #13784 
- Enables the focusing tab to take multiple files and also option of utilising two different type of focusing
- Presenter test has also been updated

_Note That_
- The `Plot Data Representation` is not compatible with the multi-run focusing yet
- `Focus Sum Of Files` will not be functioning on this branch

**To Test**
`Interface/Diffraction/Engineering Diffraction`
_Get initial settings from [here](https://github.com/mantidproject/mantid/pull/13501)_

Provide any of three focusing text field with a multiple run number for example `228061-228063`
You can find `ENGINX` inside the system & unit test data
One or various files can be renamed to `228061`, `228062` and `228061`
Make sure that the directory where files are found is mentioned in the Mantid `manage directory`.
Click Focus and you should end up with with saved files save by its own `run number` and `bank number` 

_Things to look-out for:_
Focusing should only carry out when `Focus Individual Run Files Separately` is selected 
Output files have been all generated & saved when `Output Files` is ticked
All of them `Plot Focused Workspace` for each separate run number, even though `Replacing Plot` and `Waterfall` isn't how it should be
